x-pack/plugins/security_solution/scripts/beat_docs/build.js

```diff
@@ -26,35 +26,32 @@ const zlib = require('zlib');
 
 const OUTPUT_DIRECTORY = resolve('scripts', 'beat_docs');
 const OUTPUT_SERVER_DIRECTORY = resolve('server', 'utils', 'beat_schema');
+const BEATS_VERSION = '7.9.0';
```

Comment:
I am wondering if we should update the version to `7.12.0` like described in the PR

Reply:
The fields changes were in fact from 7.12.0; I must have failed to commit that change here 👍